creator-node/src/app.js

```diff
@@ -195,17 +192,17 @@ const initializeApp = (port, serviceRegistry) => {
   app.set('audiusLibs', serviceRegistry.libs)
   app.set('blacklistManager', serviceRegistry.blacklistManager)
   app.set('trustedNotifierManager', serviceRegistry.trustedNotifierManager)
+  app.set(
+    'premiumContentAccessChecker',
+    serviceRegistry.premiumContentAccessChecker
+  )
```

Comment:
i think it should just work without this?

Reply:
yup gotchu